src/SFA.DAS.FindEmploymentSchemes.Web/Views/Shared/_Layout.cshtml

```diff
@@ -19,32 +19,23 @@
       <link rel="apple-touch-icon" sizes="167x167" href="/assets/images/govuk-apple-touch-icon-167x167.png"/>
       <link rel="apple-touch-icon" sizes="152x152" href="/assets/images/govuk-apple-touch-icon-152x152.png"/>
       <link rel="apple-touch-icon" href="/assets/images/govuk-apple-touch-icon.png"/>
-      <style asp-add-nonce>.hidden {display:none;}</style>
```

Comment:
Did you mean to make these other changes?

Reply:
No, as we discussed some code changes from other commits have been removed. The only changes I intended were the final 2 adding the cookies page link back into the cookie banner. The others can be rolled back if needed.

Reply:
Rolled back unintentional changes